app/helpers/currency_helper.rb

```diff
@@ -1,5 +1,6 @@
 module CurrencyHelper
-  def format_currency(amount)
-    Money.new(amount, AlaveteliConfiguration.iso_currency_code).format
+  def format_currency(amount, discard_zero_cents = false)
```

Comment:
We only support Ruby 2.3+ now, so a keyword argument would be compatible and make this a lot cleaner. I think it would also make sense to mirror the name the `Money` lib uses so that you don't have to translate between the two variable names.

```ruby
def format_currency(amount, no_cents_if_whole: false)
  Money.new(amount, AlaveteliConfiguration.iso_currency_code).
    format(no_cents_if_whole: no_cents_if_whole)
end
```